config.yaml

```diff
@@ -41,6 +41,25 @@ options:
       Enables the Hyper-V compute driver for clustering. This requires at least two Hyper-V compute
       joined to a cluster.
       NOTE: Available only in releases >= Mitaka
+  csv-performance-size:
+    type: string
+    default: ''
+    description: |
+      Represents the size occupied by the CSV requested from the s2d-proxy charm on the Performance
+      tier. In a Hyper-Converged scenario, each nova-hyperv unit requests a CSV to store the virtual machines.
+      Example: '20GB' or '20480MB'
```

Comment:
IMO, `GB` is more than fine. A float variable from Juju would have been better in case someone wants `0.5GB` or `10.5GB`, but we can work only with `int` as well.